app/presenters/organisation_content_presenter.rb

```diff
@@ -0,0 +1,64 @@
+class OrganisationContentPresenter < CSVPresenter
+  def initialize(scope = Artefact.where(owning_app: "publisher"))
+    super(scope)
+    self.column_headings = [
+        :id,
+        :name,
+        :format,
+        :slug,
+        :state,
+        :browse_pages,
+        :topics,
+        :organisations,
+        :need_ids
+    ]
+  end
+
+private
+
+  def latest_edition_for(artefact)
+    Edition.where(panopticon_id: artefact.id.to_s).desc(:created_at).first
+  end
+
+  def get_value(header, artefact)
+    latest_edition = latest_edition_for(artefact)
+
+    return super if latest_edition.nil?
+
+    content_id = artefact.content_id
+
+    case header
+    when :browse_pages
+      expanded_links(content_id, 'mainstream_browse_pages', 'base_path', /\/browse\//)
+    when :topics
+      expanded_links(content_id, 'topics', 'base_path', /\/topic\//)
+    when :organisations
+      expanded_links(content_id, 'organisations', 'title', /.^/)
+    when :need_ids
+      artefact.need_ids.join(',')
+    when :format
+      artefact.kind
+    else
+      super
+    end
+  end
+
+  def expanded_links(content_id, outer_key, inner_key, regex)
+    response = Services.publishing_api.get_expanded_links(content_id)
+    if response.nil? || response.raw_response_body.empty?
+      return ''
+    end
+    join_strings(response.to_hash, outer_key, inner_key, regex)
+  end
+
+  def join_strings(response, outer_key, inner_key, regex)
```

Comment:
Can we make the regex an optional argument?  It'd be nice if we didn't need to pass in the "everything matches" regex when we are fetching organisations.  It took me a moment to work out what the `/.^/` regex was for when looking at that method call, and we could avoid that effort on the part of the reader, and avoid the computational effort of doing the match if we want the whole string.

Something like the following might work:

```
def join_string(response, outer_key, inner_key, regex = nil)
  results = response.fetch('expanded_links', {}).fetch(outer_key, {})
  values = results.map { |result| results.fetch('inner_key', '') }
  values = values.map { |value| value.gsub(regex, '') } unless regex.nil?
  values.join(',') 
end
```

Might be worth renaming `regex` to something like `pattern_to_remove` or something like that so it's obvious what the regex is used for.  `regex` is really a type name that tells me what the variable is, but not what it's going to be used for and what it's going to be used for is often more helpful when reading the code.